suresh/loops.js

```diff
@@ -0,0 +1,317 @@
+//Loops for / while
+
+var arr = [1,2,3,4,5];
+
+//write a function that returns all odd numbers - [1,3,5]
+function oddNos (array) {
+  var oddNums = [];
+  var arrLength = array.length;
+
+  for (var i = 0; i < arrLength; i++) {
+ 	  if (array[i] % 2 !== 0) {
```

Comment:
Incorrect indent. (There are more in the file as well)

Reply:
updated